packages/appstore/src/components/options-multipliers-listing/index.tsx

```diff
@@ -29,13 +29,13 @@ const OptionsAndMultipliersListing = () => {
     const OptionsTitle = () => {
         if ((low_risk_cr_non_eu || high_risk_cr || cr_demo) && !isMobile()) {
```

Comment:
shouldnt we use is_mobile here

```suggestion
        if ((low_risk_cr_non_eu || high_risk_cr || cr_demo) && !is_mobile) {
```

Reply:
Updated.